src/guides/guide_changi_short.md

```diff
@@ -4,17 +4,17 @@
 
 #### 1. Open network dropdown.
 <p align="center">
-  <img width="356" alt="step1" src="https://github.com/DeFiCh/handbook/assets/113342097/ab62ae59-a140-45ff-a1e8-17db0c0a8683">
+  <img width="356" alt="step1" src="https://github.com/DeFiCh/handbook/blob/virgc/testnet_doc/src/guides/screenshots/step1_guide_changi_short.png">
 </p>
 
 #### 2. Add network
 <p align="center">
-<img width="354" alt="step2" src="https://github.com/DeFiCh/handbook/assets/113342097/64a24837-114c-406f-b624-63f4974677fa">
+<img width="354" alt="step2" src="https://github.com/DeFiCh/handbook/blob/virgc/testnet_doc/src/guides/screenshots/step2_guide_changi_short.png">
```

Comment:
Need to link to the local file. You can also remove the centre align when using this way to link images.

```suggestion
![](./screenshots/step2_guide_changi_short.png)
```